src/client.js

```diff
@@ -10,6 +10,7 @@ const Network = require("./models/network");
 const ircFramework = require("irc-framework");
 const Helper = require("./helper");
 const UAParser = require("ua-parser-js");
+const MessageStorage = require("./plugins/sqlite");
```

Comment:
Is it on purpose that `MessageStorage` == `sqlite.js`? I'm fine if so, just a sanity check in case it was forgotten with a rebase or something.

Reply:
Yeah this is correct.